artemis/src/main/java/tech/pegasys/artemis/GenesisCommand.java

```diff
@@ -60,7 +63,7 @@ public void generate(@Mixin MockGenesisParams params) throws IOException {
     try (final OutputStream fileStream =
         outputToFile ? new FileOutputStream(params.outputFile) : System.out) {
       if (outputToFile) {
-        STDOUT.log(
+        LOG.log(
```

Comment:
Do these now wind up going to stdout? `GenesisCommand` and `DepositCommand` are expected to be used interactively so we want to make sure these messages are visible to the user, not going off to a file by default.

Reply:
Ah, right. I'll add them back to the STATUS_LOG output in the next PR